Lib/test/test_posixpath.py

```diff
@@ -660,6 +660,20 @@ def test_realpath_resolve_first(self):
             safe_rmdir(ABSTFN + "/k")
             safe_rmdir(ABSTFN)
 
+    @os_helper.skip_unless_symlink
+    @skip_if_ABSTFN_contains_backslash
```

Comment:
Is it needed? How does the test work on Windows?

Reply:
It won't work on Windows because `ABSTFN` is an absolute path with backslash separators. These decorators are also used for other tests in this suite.